wom/src/main/scala/wdl4s/wom/graph/CallNode.scala

```diff
@@ -49,7 +49,9 @@ object TaskCall {
 
 object CallNode {
 
-  final case class CallWithInputs(call: CallNode, inputs: Set[GraphInputNode])
+  final case class CallWithInputs(call: CallNode, inputs: Set[GraphInputNode]) {
+    def nodes: Set[GraphNode] = Set.empty[GraphNode] + call ++ inputs
```

Comment:
I thought there was an issue with the types not working out right, but apparently not.  Will fix.